### ISSUE TITLE
Can't handle buffer reloading when files were changed outside of Vim

### ISSUE THREAD

**Issue by bootleq** [bootleq/LargeFile#3]
Vim can detect file changes outside of Vim, and try to reload the buffer.
See `help 'autoload'` and `help timestamp`.

In these cases,  BufUnload event does not occur.
BufRead, BufReadPost events do occur, and could not be at current editing buffer.

**Comment by idbrii** [comment]
FYI, I "fixed" that error by disabling s:ParenMatchOff: idbrii/LargeFile@09fdbf6bd3241aeeff2d8d72dae6a58be692e787

It occurred for me for a different reason: When opening loading 29 MB file and then a 28 MB file in a split.

    gvim 28mb.txt 29mb.txt +split +n

By this point I've given up and I'm going to give mhinz/vim-hugefile a try.

**Comment by bootleq** [comment]
Hi, I have problem reproducing error `E201` with split.
Have written some test script but can't see `E201` to happen:

```
./test/setup.sh
./test/load_large_files_in_split.sh
```

**Comment by idbrii** [comment]
You're right, on your branch the error doesn't occur for my use case (and I don't need to disable s:ParenMatchOff anymore). I haven't seen the error since I started using your branch.
